preprocess/clean/CA/Makefile

```diff
@@ -4,32 +4,29 @@
 # =========================================
 # us-post-data/preprocess/clean/CA/Make
 
-officers := input/cpra-23-3241-officers.csv
-employment := input/cpra-23-3241-employment.csv
-
 .PHONY: all clean
```

Comment:
I've generally seen `.PHONY` statements go above each individual command so you can make sure that all new commands are included.